hotosm/tasking-manager
# Include gender field in user profile

This pull request closes #1372 and changes were done within both client and server.

The following gender options were included:
- Male.
- Female.
- Prefer to self-describe
- Prefer not to say.

**How to test**

0. Execute database migrations.
1. start your client port 3000
2. Login with your osm credentials.
3. Go to your user settings.
4. Select the desired gender option.
5. Press save

**Note.** Every time some user select the third option, a text field will appear.
![image](https://user-images.githubusercontent.com/3285923/66236853-2a07c000-e6b9-11e9-9414-ec7eee7e5a74.png)
